config/locales/views/groupings/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  views:
+    groupings:
+      manage: "Manage"
+      set_team_settings: "Set of teams settings"
+      drag_and_drop_student_avatars: "You can <b>drag and drop</b> the student avatars to move them from one team to another."
+      group_has_no_teams: "\"%{group_title}\" does not have any teams."
```

Comment:
The title won't change between locales, so can the string just be "does not have any teams."? If we ever decide to change that variable name, or some sort of logic around it, ideally we don't have to go combing through locale files to find usages.